man/fcase.Rd

```diff
@@ -54,5 +59,12 @@ fcase(
 	x > 5L, 3L,
 	default = 5L
 )
+
+# fcase can be used for recursion, unlike fifelse
+gcd_dt = function(x,y) {
```

Comment:
i guess this is greatest common denominator? can you please add a comment to explain?